libs/src/services/dataContracts/userReplicaSetManagerClient.js

```diff
@@ -100,11 +100,15 @@ class UserReplicaSetManagerClient extends ContractClient {
   async getUserReplicaSet (userId) {
     const method = await this.getMethod('getUserReplicaSet', userId)
     let currentWallet = this.web3Manager.getWalletAddress()
-    return method.call({ from: currentWallet })
+    let resp = await method.call({ from: currentWallet })
+    return {
+      primaryId: parseInt(resp.primaryId),
+      secondaryIds: resp.secondaryIds.map(x => parseInt(x))
+    }
```

Comment:
this is fine but why do this? i think this breaks contract client pattern. is this just bc the og response has the numbers as strings?

Reply:
seems to follow the other datacontract clients - https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/services/dataContracts/trackFactoryClient.js#L45. 

@dmanjunath found this discrepancy and is only an issue in dapp environment not mad dog - not sure why the two are different though